nss-cohort-36/bangazon-api-virtual_magicians
# fixed payment_type model

# Description
Changed payment_type to snake case and fixed delete payment_type function
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-shop`
4. `python manage.py makemigrations virtualmagicians`
5. `python manage.py migrate`

# Checklist:
